scripts/orchestration/agent_consistency_loader.py

```
@@ -0,0 +1,154 @@
+"""Load agent name sets from AGENT_INVENTORY, AGENT_CAPABILITY_MATRIX, and routing graph.
+
+Used by check_agent_consistency.py to enforce: routing ⊆ inventory ⊆ capability.
+Canonical paths: docs/orchestration/AGENT_INVENTORY.md, AGENT_CAPABILITY_MATRIX.md, AGENT_ROUTING_GRAPH.md.
+"""
+
+from __future__ import annotations
+
+import re
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Iterable, List, Optional, Set, Tuple
```

Flake8 (7.3.0)

[error] 'typing.Iterable' imported but unused

(F401)